repository/role_perm_postgres.go

```diff
@@ -0,0 +1,171 @@
+package repository
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/sirupsen/logrus"
+	auth_proto "stlab.itechart-group.com/go/food_delivery/authorization_service/GRPC"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/model"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/pkg/logging"
+)
+
+type RolePermPostgres struct {
+	logger logging.Logger
+	db     *sql.DB
+}
+
+func NewRolePermPostgres(db *sql.DB, logger logging.Logger) *RolePermPostgres {
+	return &RolePermPostgres{db: db, logger: logger}
+}
+
+func (r *RolePermPostgres) GetRoleById(id int) (*model.Role, error) {
+	transaction, err := r.db.Begin()
+	if err != nil {
+		logrus.Errorf("GetRoleById: can not starts transaction:%s", err)
+		return nil, fmt.Errorf("GetRoleById: can not starts transaction:%w", err)
+	}
+	var role model.Role
+	query := "SELECT id, name FROM roles WHERE id = $1"
+	row := transaction.QueryRow(query, id)
```

Comment:
Is it any reason for using transaction here? I would say that usually you don't need explicit transaction when you do just one get request.